inst/json/metadata_0.1.0.json

```
@@ -86,6 +86,10 @@
       { "id": "life_years_working_age", "label": "Working-age adults" },
       { "id": "life_years_retirement_age", "label": "Retirement-age adults" }
     ],
+    "metrics": [
+      { "id": "usd", "label": "USD" },
+      { "id": "life_years", "label": "life years" }
+    ],
```

```suggestion
    "metrics": [
      { "id": "usd_millions", "label": "USD" },
      { "id": "life_years", "label": "life years" }
    ],
```

I think it would be useful to record the scale of the metric in the id like this.